internal/api/admin.go

```diff
@@ -352,6 +352,10 @@ func (a *API) adminUserCreate(w http.ResponseWriter, r *http.Request) error {
 		params.Password = &password
 	}
 
+	if err := a.checkPasswordStrength(ctx, *params.Password); err != nil {
+		return err
+	}
```

Comment:
This alters the behavior of `adminUserCreate`, as previously password strength was not checked in this function. If the intention is to allow an admin to create any password they chose, then it might be better to just add a seperate check for length here instead of using `checkPasswordStrength`.

Reply:
yeah that makes sense, i realised that there's a better place to do this check - we can check for the password length before generating the hash - https://github.com/supabase/gotrue/blob/42802886c5e11b82ae51fa4f1ec6994c3b29d521/internal/crypto/password.go#L73

wdyt?

Reply:
I think this is a great suggestion that avoids having to repeat this length check separately. Let's do it this way.